examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryTornadoesIT.java

```diff
@@ -41,7 +41,7 @@
  *
  * <pre>
  *  ./gradlew integrationTest -p examples/java/ -DintegrationTestPipelineOptions='[
- *  "--tempLocation=gs://your-location/"]'
+ *  "--tempLocation=gs://apache-beam-testing-developers/"]'
```

Comment:
Did you mean to do this update ?

Reply:
yes, this is intentional. I wanted to include a command you could copy and paste, without needing to spend 5 minutes locating  bucket parameter. I think we should check this in